cqrs-example/cqrs-example.cabal

```diff
@@ -24,12 +24,12 @@ Executable cqrs-example
   Main-is:           CQRSExample/Main.hs
   Ghc-options:       -Wall
   Build-depends:     base >= 4.8 && < 5
-                   , aeson >= 0.8 && < 0.9
+                   , aeson >= 0.8
```

Comment:
I'm not big on removing upper bounds just for stack. What versions are you using?

(I'd be happy to bump the lower bound + upper-bound as long as the thing still builds with cabal-install, obviously.)